LICENSE.md

```diff
@@ -42,4 +42,5 @@ rickyboy320 <rickw320@hotmail.com>
 DoNotSpamPls <7570108+DoNotSpamPls@users.noreply.github.com>
 JRoy <joshroy126@gmail.com>
 Robert Norman <spottedleaf@spottedleaf.dev>, <Spottedleaf@users.noreply.github.com>
+Callum Seabrook <callum.seabrook@prevarinite.com>
```

Comment:
```suggestion
```

Reply:
What's the issue with that? I thought you just had to put your name in there to license your stuff under MIT... or am I missing something.

Reply:
YOU. CANNOT. LICENSE. YOUR. CHANGES. TO. THE. README. IT'S. ONLY. PATCHES.

Reply:
Yours truly licensing 10 lines of generic important statements

Reply:
Okay, okay, I'll change it hold on. Didn't know, my apologies.